### PR TITLE
Site search tweaks

### PR DESCRIPTION
**Pull request by neilblair**
Fixes issue where search input is empty after performing a search.

### CODE REVIEW

File: config/sync/block.block.footer.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies:
 id: footer
 theme: nicsdru_dept_theme
 region: footer
-weight: -25
+weight: -23
```
Comment (ooneill547) [inline config/sync/block.block.footer.yml:14]: I don't think this should be changed

**Review by ooneill547 — APPROVED** [review]
Just need to remove the weight change in the footer

File: config/sync/block.block.footer.yml (hunk 1 above)
Comment (omahm) [reply config/sync/block.block.footer.yml:14]: Is this to pop the footer links (copyright etc) above the department links?

Comment (neilblair) [reply config/sync/block.block.footer.yml]: Oopsy fixed ☝🏻

**Pull request merged**